wgsl/index.bs

```diff
@@ -8769,6 +8692,28 @@ WebGPU provides no guarantees about:
     the invocations of another workgroup.
     That is, you cannot assume that workgroups are launched in a particular order.
 
+## Fragment Shaders and Helper Invocations ## {#fragment-shaders-helper-invocations}
+
+Invocations in the [=fragment shader stage=] are divided into 2x2 grids of neighbouring invocations.
```

Comment:
Should say in what sense they are "neighbouring".

Suggest

>  of invocations with neighboring framebuffer positions in the X and Y dimensions.

(I filed separate issue #3177 so we can make 'position' a cross-reference)

Reply:
I rebased on top of the definitions and made this a link.